interface/wx/spinctrl.h

```diff
@@ -145,6 +145,13 @@ class wxSpinCtrl : public wxControl
     */
     int GetValue() const;
 
+    /**
+        Get the value f increment fr a spin control.
+
```

Comment:
It would probably be useful to mention that the default value is 1 and add `@see SetIncrement()`.